### ISSUE TITLE
Keyboard navigation doesn't work as expected in list views

### ISSUE THREAD

**Issue by oSoMoN** [canonical/ubuntu-desktop-installer#343]
I'm testing the desktop installer without a mouse, and keyboard navigation doesn't work as I'd expect: when I press tab in a list view, I would expect the next UI element (e.g. the "Next" button on the welcome screen) to become focused, instead tab goes through all the elements in the list view, before jumping to the next button.

This isn't so bad on the welcome screen because the list of languages is fairly short, but it becomes terrible on the keyboard layout selection screen, where I cannot give keyboard focus to the navigation buttons before going through all the keyboard layouts and variants one by one.

**Comment by jpnurmi** [comment]
I have reported a bunch of focus traversal issues to Flutter:
  - [Cannot tab out from a long ListView](https://github.com/flutter/flutter/issues/91748)
  - [Directional focus traversal stops working after scrolling ListView](https://github.com/flutter/flutter/issues/91741)
  - [Focus traverses cached offscreen ListView items](https://github.com/flutter/flutter/issues/91795)
  - [Tab focus traversal doesn't scroll an uncached ListView](https://github.com/flutter/flutter/issues/91743)

**Comment by madsrh** [comment]
Slightly related, hitting a key in the list, doesn't jump to that letter. Eg. if English keyboard is selected, pressing D should jump to Danish

![image](https://user-images.githubusercontent.com/3986894/204086767-b21ca6a3-d105-486b-9749-ec749f390a62.png)
